iFixit/pulldasher
# Docker Compose: Make DB initialization idempotent

There's an issue where you can't run `docker-compose` on an already
initialized db, because the `migrate` script tries to create tables that
already exist. By modifying the `dump-schema` command to use `CREATE
TABLE IF NOT EXIST` statements, bringing up pulldasher instance with and
already initialized db with `docker-compose` will work without failure.

This also moves some one-off scripts into the `bin` directory so that
the app root directory looks a little less cluttered.

### QA

Bring up a pulldasher instance using `docker-compose`, make sure it starts without issue.
Kill the instance and then bring it up again and make sure it still starts without issue.
